tests/pester-tests/commands/choco-install.Tests.ps1

```diff
@@ -1,4 +1,4 @@
-﻿Describe "choco install" -Tag Chocolatey, InstallCommand {
+Describe "choco install" -Tag Chocolatey, InstallCommand {

```

Comment:
It looks like this encoding changed... not entirely sure why though.

Reply:
I am not sure why this changed from UTF8 with BOM, nor why some of the files around this one don't have a BOM. However, the vscode setting for powershell files should be enforcing this with a BOM, so I've returned it to UTF8 with a BOM.